pkgs/development/libraries/librsvg/default.nix

```diff
@@ -78,7 +79,8 @@ stdenv.mkDerivation rec {
 
     "--enable-installed-tests"
     "--enable-always-build-tests"
-  ] ++ lib.optional stdenv.isDarwin "--disable-Bsymbolic";
+  ] ++ lib.optional stdenv.isDarwin "--disable-Bsymbolic"
+    ++ lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "RUST_TARGET=${rust.toRustTarget stdenv.hostPlatform}";
```

Comment:
Would it make sense for `rustc` to set this through a hook? The `zcash` package does the same.

cc rustc maintainers @madjar @cstrahan @globin @havvy

Reply:
`RUST_TARGET` does not seem to be a standard envvar per https://doc.rust-lang.org/cargo/reference/environment-variables.html, most likely to be specific to these packages.

Reply:
https://gitlab.gnome.org/search?project_id=296&search=RUST_TARGET